src/wiki/plugins/images/models.py

```diff
@@ -119,15 +119,35 @@ def on_image_revision_delete(instance, *args, **kwargs):
     if not instance.image:
         return
 
-    # Remove image file
+    # from attachments.models.py (line 166) :
+    # Remove file
+    path = instance.image.path.split("/")[:-1]
```

Comment:
This won't work on non-POSIX systems.... I know one such commercial operating system that's quite popular and could be affected :smile: 

```suggestion
    path = os.path.dirname(instance.image.path)
```

Reply:
ok. that same must apply to the 'attachements' plugin too then?
I have no way of testing Windows and/or OSX

Reply:
It's okay to not test on these exotic operating systems as long as we use `os.path.split()` without specifying the path separator or use `os.path.dirname` since it's also platform agnostic.

Reply:
If you press "Commit suggestion" on my suggestion, it will go strait in, you can also add it manually.

Reply:
Okay, seeing that the previous code supports backends (such as S3 storage), I'm wondering if you can implement this in a way that doesn't introduce new regressions. Others have opened issues and worked on these storage backends, so we shouldn't break that, even though tests might not cover it.